spec/pipelines/pacbio/sample_sheet_spec.rb

```diff
@@ -61,7 +61,7 @@
       column = sample_sheet_configuration.columns.children['Plate 1']
       expect(column).to be_present
       expect(column.fetch('type')).to eq(:model)
-      expect(column.fetch('value')).to eq('plate.run.sequencing_kit_box_barcode')
+      expect(column.fetch('value')).to eq('plate.sequencing_kit_box_barcode')
       expect(column.fetch('populate')).to eq(populate)
```

Comment:
Could `plate_number` be added to? Even thought its just 1 for all, until the next story is implemented